### PR TITLE
Updated README with sunset info.

### PR DESCRIPTION
**Pull request by arjuhe**
- Added sunset date
- Added notice of upcoming role changes
- Added notice of upcoming installer changes
- Fixed up linting issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +11,15 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
 #############################################################################
 ```
+---
+Upcomming changes:
+- Crucible's roles have been migrated to the [Red Hat CI OCP Ansible Collection](https://github.com/redhatci/ansible-collection-redhatci-ocp) available for download at [Ansible Galaxy](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/).  
+- Crucible's playbooks will continue to work but as always, you are encouraged to fork crucible and maintain your own playbooks.
+- On <ins>Monday April 18th</ins> Crucible will crossover and default to using the *redhatci.ocp collection*. At this point all roles will be removed from `roles/`.
```
Comment (nocturnalastro) [inline README.md:18]: most roles (there are two that we will be putting back)

File: README.md [hunk 2]
```diff
@@ -19,6 +11,15 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
 #############################################################################
 ```
+---
+Upcomming changes:
+- Crucible's roles have been migrated to the [Red Hat CI OCP Ansible Collection](https://github.com/redhatci/ansible-collection-redhatci-ocp) available for download at [Ansible Galaxy](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/).  
+- Crucible's playbooks will continue to work but as always, you are encouraged to fork crucible and maintain your own playbooks.
```
Comment (nocturnalastro) [inline README.md:17]: Perhaps put out a call to anyone who wants to main them here?

File: README.md (hunk 1 above)
Comment (arjuhe) [reply README.md:18]: Validate and what else is staying? You mentioned it earlier but I couldn't pay attention.

File: README.md (hunk 2 above)
Comment (arjuhe) [reply README.md:17]: I kept adding and removing the call for stewardship. I'll put it back.

File: README.md (hunk 1 above)
Comment (nocturnalastro) [reply README.md:18]: The display plan

**Pull request merged**